packages/victory-voronoi/package.json

```diff
@@ -28,7 +29,170 @@
     "react": ">=16.6.0"
   },
   "scripts": {
-    "version": "yarn run nps build-libs && nps build-dists"
+    "build": "wireit",
+    "build:lib": "wireit",
+    "build:lib:esm": "wireit",
+    "build:lib:cjs": "wireit",
+    "build:dist": "wireit",
+    "build:dist:dev": "wireit",
+    "build:dist:min": "wireit",
+    "types:check": "wireit",
+    "types:create": "wireit",
+    "format": "wireit",
+    "lint": "wireit",
+    "jest": "wireit"
   },
-  "sideEffects": false
+  "wireit": {
+    "build": {
+      "dependencies": [
```

Comment:
So here's my mental-model for our "dependency graph":
```mermaid
flowchart
    subgraph Key
    TLS{{entry scripts}} --> DS[Dependent scripts]
    end

    lint{{lint}}
    types:check{{types:check}}
    storybook{{storybook}}
    jest{{jest}}
    types:create{{types:create}}
    build{{build}}

    lint --> deps:types:create
    types:check --> deps:types:create
    types:create --> deps:types:create

    build --> types:create
    build --> build:lib:esm
    build --> build:dist
    build --> build:lib:cjs

    build:dist --> build:lib:cjs


    storybook -->  build:lib:cjs
    jest --> build:lib:cjs
```

Reply:
Pretty close!!!

Storybook is webpack based, which means dependent on `build:lib:esm`. Similarly `build:dist` is webpack and also `:esm`

Jest is the only one using `build:lib:cjs`.